### PR TITLE
DS-558: fontWeights tokens

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/Rating/index.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const Rating: FC<RatingProps> = ({
           ))}
         </div>
       </div>
-      <p className="mt-rating-marginSupportingText text-rating-base-supportingText text-rating-supportingText empty:hidden">
+      <p className="text-rating-supportingText font-rating-weightSupportingTexr mt-rating-marginSupportingText text-rating-base-supportingText empty:hidden">
```
Comment (monikaracka) [inline packages/design-system/src/components/Rating/index.tsx:146]:
```suggestion
      <p className="text-rating-supportingText font-rating-weightSupportingText mt-rating-marginSupportingText text-rating-base-supportingText empty:hidden">
```

File: packages/design-system/src/components/Review/index.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const List: FC<ListProps> = ({ list }) => {
     <ul className="flex flex-col gap-1">
       {list.map((item, index) => (
         <li className="text-review-base-listItem" key={`${item.text}-${index}`}>
-          <span className="text-review-base-listItemLabel font-medium">
+          <span className="font-medium text-review-base-listItemLabel">
```
Comment (monikaracka) [inline packages/design-system/src/components/Review/index.tsx:16]: missing token

File: packages/design-system/src/components/Stepper/index.tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ const StepperComponents = {
         </p>
         <p
           className={classNames(
-            "text-stepper-fontDescription empty:hidden",
+            "text-stepper-fontDescription font-stepper-weightDescriptio empty:hidden",
```
Comment (monikaracka) [inline packages/design-system/src/components/Stepper/index.tsx:100]:
```suggestion
            "text-stepper-fontDescription font-stepper-weightDescription empty:hidden",
```

File: packages/design-system/src/components/Toast/index.tsx [hunk 4]
```diff
@@ -92,8 +92,9 @@ export const ToastComponents = {
 
   Text: ({ children, textType, className }: TextToastProps) => {
     const textTypeVariants: GenericKeys = {
-      title: "text-toast-fontTitle font-medium",
-      description: "text-toast-fontDescription font-normal",
+      title: "text-toast-fontTitle font-toast-weightTitle",
+      description:
+        "text-toast-fontDescription font-normal font-toast-weightDescription",
```
Comment (monikaracka) [inline packages/design-system/src/components/Toast/index.tsx:97]:
```suggestion
        "text-toast-fontDescription font-toast-weightDescription",
```

File: packages/design-system/src/helpers/tokens/checkbox.json [hunk 5]
```diff
@@ -169,6 +179,26 @@
       }
     }
   },
+  "weightOptionalLabelText": {
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/checkbox.json:182]: Put it in order for token types. Ideally, we should have the same order in each JSON with tokens.

File: packages/design-system/src/helpers/tokens/datePicker.json [hunk 6]
```diff
@@ -328,6 +337,26 @@
...
+  "weightHeader": {
+    "text": {
+      "fontWeight": {
+        "default": {
+          "value": "{regular}",
+          "type": "fontWeights"
+        }
+      }
+    }
+  },
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/datePicker.json:359]: Same issue with the order here.

File: packages/design-system/src/helpers/tokens/dropdown.json [hunk 7]
```diff
@@ -369,6 +379,16 @@
...
+  "weightOptionLabel": {
+    "text": {
+      "fontWeight": {
+        "default": {
+          "value": "{medium}",
+          "type": "fontWeights"
+        }
+      }
+    }
+  },
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/dropdown.json:391]: Same issue with the order here.

File: packages/design-system/src/helpers/tokens/featuredText.json [hunk 8]
```diff
@@ -73,6 +93,16 @@
...
+  "weightUrl": {
+    "text": {
+      "fontWeight": {
+        "default": {
+          "value": "{medium}",
+          "type": "fontWeights"
+        }
+      }
+    }
+  },
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/featuredText.json:105]: Same issue with the order here.

File: packages/design-system/src/helpers/tokens/tokensConfigs/fontWeights.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "medium": {
+    "value": "500",
+    "type": "fontWeights"
+  },
+  "bold": {
+    "value": "600",
+    "type": "fontWeights"
+  }
+}
```
[the remark this answers is not in the document]
Comment (zuzannanap) [reply packages/design-system/src/helpers/tokens/tokensConfigs/fontWeights.json:14]: yeah it's a good idea, i changed it

File: packages/design-system/src/helpers/tokens/accordion.json [hunk 10]
```diff
@@ -168,5 +168,35 @@
...
+          "type": "fontSizes"
+        }
+      }
+    }
+  },
+  "weightTitle": {
+    "text": {
+      "fontWeight": {
+        "default": {
+          "value": "{semi-bold}",
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/accordion.json:186]: In config is semibold.

File: packages/design-system/src/helpers/tokens/tokensConfigs/fontWeights.json [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "medium": {
+    "value": "500",
+    "type": "fontWeights"
+  },
+  "semibold": {
+    "value": "600",
+    "type": "fontWeights"
+  }
+}
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/tokensConfigs/fontWeights.json:14]: I meant we should have each `font weight` (100, 200, 300,  ..., 900) in config.

**Review by monikaracka — CHANGES_REQUESTED** [review]

File: packages/design-system/src/helpers/defaultTheme.js [hunk 12]
```diff
@@ -11,6 +12,8 @@ const spacings = require("./tokens/tokensConfigs/spacings.json");
 const colorPaletteConfig = colorTokens.getColorsConfig(colors);
 const borderRadiusConfig = radiusTokens.getRadiusConfig(borderRadius);
 const fontSizesConfig = fontSizeTokens.getFontSizesConfig(fontSizeTokens);
+const fontWeightsConfig =
+  fontWeightsTokens.getFontWeightsConfig(fontWeightsTokens);
```
Comment (monikaracka) [inline packages/design-system/src/helpers/defaultTheme.js:16]: You didn't add this to config.

**Review by monikaracka — APPROVED** [review]

**Pull request merged**